### PR TITLE
Add getting started cards to link new docs

### PR DESCRIPTION
**Pull request by bitsondatadev**
Add three top-level cards to the Getting Started page that give the page a more _Getting Started_ feel.
Screen Shot 2022-10-18 at 2 07 05 PM

### CODE REVIEW

**Review by mosabua — CHANGES_REQUESTED** [review]
Just some initial feedback

* Try to find one line titles
* Aim to have length of description so that boxes and buttons align
* Avoid same title on each button

**Comment by bitsondatadev** [comment]
> Just some initial feedback
> 
> * Try to find one line titles
> * Aim to have length of description so that boxes and buttons align
> * Avoid same title on each button

Addressed

**Review by mosabua — COMMENTED** [review]
Looks good to me. The text in each box is a bit wordy. Not sure if this is okay with @martint and other maintainers as is, or if we should make it shorter.

File: download.md [hunk 1]
```diff
@@ -6,6 +6,53 @@ show_hero: true
 ---
 
 <div class="container">
+   <!-- Card deck -->
+    <div class="card-deck spacer-30">
+    <div class="card mb-4">
```
Comment (mosabua) [inline download.md:11]: is it really mb-4 .. not md-4 ?

**Comment by mosabua** [comment]
Also would be good to get some feedback/ideas from @aliloney

File: download.md (hunk 1 above)
Comment (bitsondatadev) [reply download.md:11]: I'm just following suite to the other cards just beneath.

**Comment by aliloney** [comment]
This looks great! Could we shorten the paragraphs a bit and make sure there isn't just a single word left behind on the last line? Otherwise, awesome :)

File: download.md (hunk 1 above)
Comment (aliloney) [reply download.md:11]: This is right

**Review by aliloney — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
Looks like there are at least two votes for shortening the paragraph @bitsondatadev  .. can you go for some word slashing?

File: download.md [hunk 2]
```diff
@@ -6,6 +6,50 @@ show_hero: true
...
+              or running full-scale systems on the cloud.
+            </p>
+            <a class="btn btn-pink" href="docs/current/installation/kubernetes.html">
+                Try Trino on Kubernetes >>
+            </a>
+        </div>
+    </div>
+    <div class="card mb-4">
+        <div class="card-body text-center">
+            <h3 class="card-header-title mb-3">Deploy a Trino container</h3>
```
Comment (mosabua) [inline download.md:27]:
```suggestion
            <h3 class="card-header-title mb-3">Run a Trino container</h3>
```

File: download.md [hunk 3]
```diff
@@ -6,6 +6,50 @@ show_hero: true
...
+            <p class="card-text">Run Trino using container tools like
```
Comment (mosabua) [inline download.md:28]: Start Trino

File: download.md [hunk 4]
```diff
@@ -6,6 +6,50 @@ show_hero: true
...
+              <a href="https://docs.docker.com/get-started/overview/">Docker</a>.
+              Use this method to get started without worrying about scalability
```
Comment (mosabua) [inline download.md:30]:
```suggestion
              Use this method to get experiment without worrying about scalability
```

File: download.md [hunk 5]
```diff
@@ -6,6 +6,50 @@ show_hero: true
...
+            </p>
+            <a class="btn btn-pink" href="docs/current/installation/containers.html">
+                Spin up Trino on Docker >>
+            </a>
+        </div>
+    </div>
+    <div class="card mb-4">
+        <div class="card-body text-center">
+            <h3 class="card-header-title mb-3">Deploy Trino manually</h3>
+            <p class="card-text">Use this method for fine-tuned deployments that
```
Comment (mosabua) [inline download.md:41]:
```suggestion
            <p class="card-text">Use the tarball for fine-tuned deployments and
```

File: download.md [hunk 6]
```diff
@@ -18,8 +62,11 @@ show_hero: true
     <div class="card mb-4">
```
Comment (mosabua) [inline download.md:62]: We need a Downloads header or so here

File: download.md [hunk 7]
```diff
@@ -18,8 +62,11 @@ show_hero: true
...
         <div class="card-body text-center">
             <h3 class="card-header-title mb-3">Server package</h3>
-            <p class="card-text">See <a href="docs/current/installation/deployment.html">Deploying 
-            Trino</a> for complete deployment instructions.</p>
+            <p class="card-text">
+              See <a href="docs/current/installation/deployment.html">Deploying Trino</a>
```
Comment (mosabua) [inline download.md:66]: Make this shorter since it duplicates stuff from above partially now

**Review by mosabua — CHANGES_REQUESTED** [review]

File: download.md (hunk 7 above)
Comment (mosabua) [reply download.md:66]: Ping @bitsondatadev ?

File: download.md (hunk 6 above)
Comment (mosabua) [reply download.md:62]: Ping @bitsondatadev ?

File: download.md [hunk 8]
```diff
@@ -6,8 +6,53 @@ show_hero: true
...
+                Learn how to deploy Trino >>
+            </a>
+        </div>
+    </div>
+    <!-- Card -->
+    </div>
+    <!-- Card deck -->
   <div class="row spacer-60">
     <div class="col-md-12 text-center">
+       <h2>Downloads</h2>
```
Comment (mosabua) [inline download.md:55]: The spacing here is off .. the gap to the cards is too big compared to the gap above the title .. not sure how to adjust

**Review by mosabua — APPROVED** [review]
Looks good now
